utils/config-utils.js

```diff
@@ -1,10 +1,30 @@
-// `CUTTLE_ENV` is needed to test prod specific files without actually putting the app in to
-// a production state, which requires an actual database connection
-const isProd =
-  process.env.CUTTLE_ENV !== undefined
-    ? process.env.CUTTLE_ENV === 'production'
-    : process.env.NODE_ENV === 'production';
-
-module.exports = {
-  isProd,
-};
+import devtools from '@vue/devtools'
+
+import { version } from '_/package.json';
+
+export function initCuttleGlobals(app) {
+  if (!window) {
+    console.error('No window object found');
+  }
+
+  const test = window.Cypress != null;
+  const cuttle = {
+    version,
+    // Expose app for debugging/testing in lower envs
+    app: window.Cypress ? app : null,
+    test,
+  };
+
+  window.cuttle = cuttle;
+
+  // Connect the devtools -- non-prod only
+  if (!import.meta.env.PROD) {
+    try {
+      devtools.connect(null, 8098);
+    } catch(err) {
+      console.warn('Failed to connect vue devtools. Try running npm run start:devtools');
+      return;
+    }
```

Comment:
Could remove this try/catch b/c it doesn't seem to catch anything

Reply:
@seriouslysean think it's better to leave it defensively? Overkill if it doesn't address the key issue when the devtools aren't open

Reply:
I don't see any issue leaving this here, don't think you need to return though.

Reply:
The idea of the return was so that we could assume success if we didn't hit the catch block. That's why the log says connected rather than connecting. It didn't really work though because connect doesn't actually error

Reply:
I just moved the return up to the try. Can probably take another look at this eventually, not sure the best way to be safe with calling the devtools script.